### PR TITLE
Changes to support ClickHouse on Z(s390x) platform

### PR DESCRIPTION
**Pull request by vibhaKulka**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Various fixes to support running ClickHouse on IBM s390x systems.

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
The pull request title "S390x support for ClickHouse" is not descriptive enough.

**Comment by alexey-milovidov** [comment]
The pull request changelog
> Embedded Keeper will always start in the background allowing ClickHouse to start without achieving quorum.

is not only incorrect, but it is mocking our team.

File: contrib/librdkafka-cmake/config.h.in [hunk 1]
```diff
@@ -66,7 +66,7 @@
 #cmakedefine WITH_SASL_OAUTHBEARER 1
 #cmakedefine WITH_SASL_CYRUS 1
 // crc32chw
-#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__)
+#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__) && !defined(__s390x__)
 #define WITH_CRC32C_HW 1
```
Comment (alexey-milovidov) [inline contrib/librdkafka-cmake/config.h.in:70]: Does s390x CPU have CRC32-C instruction?
I remember you told me the opposite: https://github.com/ClickHouse/ClickHouse/pull/40948

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]

**Comment by vibhaKulka** [comment]
> The pull request title "S390x support for ClickHouse" is not descriptive enough.

Updated the title. Thank you

File: contrib/librdkafka-cmake/config.h.in (hunk 1 above)
Comment (vibhaKulka) [reply contrib/librdkafka-cmake/config.h.in:70]: @alexey-milovidov I am checking with our compiler team on this

**Comment by rschu1ze** [comment]
@vibhaKulka Friendly ping - what's the status here? Thanks.

**Comment by rschu1ze** [comment]
Closing due to inactivity. Please re-open in case of sustained interest.

**Pull request closed without merging**